docs/docs/tutorials/view-kubernetes-logs.mdx

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 110
+title: View Kubernetes Logs
+---
+
+# View Kubernetes Logs in PRs
+
+Stoat can help you easily view and access logs for Kubernetes clusters ((Minikube)[https://minikube.sigs.k8s.io/docs/], (k3s)[https://k3s.io/], etc.) in your GitHub workflows from a comment on your pull request.
+
+Here's an example of a Stoat comment that links to an interactive log viewer for Kubernetes:
+<img src="https://stoat-dev--static.stoat.page/kubernetes.gif" alt="kubernetes gif" width={600} />
+
+## Example Repo
+
+See a real build that links to a log viewer for Kubernetes logs [this pull request](https://github.com/stoat-dev/kubernetes-log-export-action/pull/1).
+
+# Adding Kubernetes log previews to the Stoat comment
+
+In this tutorial we will show how to add a Kubernetes log preview to the Stoat comment.
+
+1. Set up Stoat with our [getting started guide](../installation).
+
+2. Add both an action to dump logs to some directory using `stoat-dev/kubernetes-log-export-action@v1` the [Stoat Action](https://github.com/stoat-dev/stoat-action) to upload the log outputs to Stoat.
+
+    ```yaml title=".github/workflows/kubernetes.yaml"
```

Comment:
Contents under the bullet points should have some indentation, so that they will appear to be part of the bullet point. Otherwise, the layout looks weird.

Reply:
I just re-added